extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-rules.service.ts

```diff
@@ -88,7 +88,8 @@ export class DragAndDropSortInputRulesService {
 
   HasElementXAtPositionY(
       answer: DragAndDropAnswer,
-      inputs: DragAndDropHasElementXAtPositionYRuleInputs): boolean {
+      inputs: DragAndDropHasElementXAtPositionYRuleInputs
+  ): boolean | undefined {
```

Comment:
I think this function should not return `undefined`, there should be some default return after a line 98 probably `return false`.

Reply:
Oh okay, done